sopel_modules/rat_board.py

```diff
@@ -43,8 +43,8 @@
 from ratlib import timeutil, starsystem
 from ratlib.api.props import SystemNameProperty
 from ratlib.autocorrect import correct
-from ratlib.api.props import *
-from ratlib.api.names import *
+from ratlib.api.props import TrackedBase, TrackedProperty, DateTimeProperty, SetProperty, ListProperty, TypeCoercedProperty, InstrumentedProperty
+from ratlib.api.names import callapi, require_permission, Permissions, getRatName, getRatId, removeTags, flushNames
```

Comment:
```suggestion
from ratlib.api.names import callapi, require_permission, Permissions, getRatName, getRatId, removeTags, flushNames
import logging
```
just gunna sneak a logging import in here to make later suggestions work 😇